### PR TITLE
vulkan-headers: bump revision

### PR DESCRIPTION
**Pull request by pthariensflame**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

The tag was updated in place to correct an error; pick up the correction!

### CODE REVIEW

File: Formula/vulkan-headers.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class VulkanHeaders < Formula
   url "https://github.com/KhronosGroup/Vulkan-Headers/archive/v1.3.248.tar.gz"
   sha256 "e2a3ef209deee3e47e67f3e57846f343c53b96fbe854f5c7427f0bfee478907f"
```
Comment (carlocab) [inline Formula/vulkan-headers.rb:5]: If the tag was updated in place, then shouldn't the checksum change?

Comment (pthariensflame) [reply Formula/vulkan-headers.rb:5]: That is a good point, and I'll correct that!  I'm not sure what I was thinking.

**Comment by pthariensflame** [comment]
I'm also not sure how many other formulae we should bump in response to this; I know at least `vulkan-loader` and my upcoming `vulkan-tools` were affected.

**Comment by pthariensflame** [comment]
Well, it's failing for the exact reason it needed this bump. XD
The upstream issue is https://github.com/KhronosGroup/Vulkan-Headers/issues/395.

**Review by carlocab — DISMISSED** [review]
Thanks!

**Pull request merged**